virt/virtual_machines/vm_networking/virt-connecting-vm-to-service-mesh.adoc

```diff
@@ -0,0 +1,19 @@
+[id="virt-connecting-vm-to-service-mesh"]
+= Connecting virtual machines to a service mesh
+include::modules/virt-document-attributes.adoc[]
+:context: virt-connecting-vm-to-service-mesh
+
+toc::[]
+
+{VirtProductName} is now integrated with OpenShift Service Mesh. You can monitor, visualize, and control traffic between pods that run virtual machine workloads on the default pod network with IPv4.
+
+== Prerequisites
+
+* You must have xref:../../../service_mesh/v2x/installing-ossm.adoc#installing-ossm[installed the Service Mesh Operator] and xref:../../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-create-smcp[deployed the control plane].
```

Comment:
Should "control plane" be more specific "ServiceMeshControlPlane" so as not to confuse with the main OCP control plane?